core/domain/certificate_assessment_services_test.py

```
@@ -195,6 +200,935 @@ def test_get_update_and_delete_certificate_assessment_offering(
+        ), mock.patch.object(
+            skill_models.SkillModel,
+            'get_multi',
+            return_value=[None, skill_1, skill_2, skill_unclassified],
+        ), mock.patch.object(
+            skill_fetchers,
+            'get_skill_from_model',
+            side_effect=lambda skill_model: skill_model,
+        ), mock.patch.object(
+            question_services,
+            'get_question_skill_links_of_skill',
+            side_effect=lambda skill_id, description: links_by_skill_id[
```

Ruff (0.16.1)

[warning] Unused lambda argument: `description`

(ARG005)